db/migrations/0005.sql

```diff
@@ -1,9 +1,32 @@
 -- +migrate Down
+DROP VIEW IF EXISTS vw_batch_num_gaps;
 ALTER TABLE data_node.offchain_data DROP COLUMN IF EXISTS batch_num;
 
 -- +migrate Up
 ALTER TABLE data_node.offchain_data
     ADD COLUMN IF NOT EXISTS batch_num BIGINT NOT NULL DEFAULT 0;
 
+-- Ensure batch_num is indexed for optimal performance
```

Comment:
Decided to modify an existing migration since it was added in the previous PR and was not deployed anywhere yet. It should be ok in this particular case.